src/server/api/medias/musicres.js

```diff
@@ -34,4 +35,26 @@ router.get('/subjectareas', (req, res) => {
   );
 });
 
+router.get('/analytics', async (req, res) => {
+  if (!CheckRoleServices.isInstructorOrAdmin(res.locals.token.roles)) {
```

Comment:
@maxpluez I see this code similar to what’s in Bruincast.

Reply:
I don't think we should be modularizing route functions? I don't see a good place to modularize it..